src/components/carousel/slide/slide.component.tsx

```diff
@@ -2,14 +2,14 @@
 import React from "react";
 import tagComponent from "../../../__internal__/utils/helpers/tags/tags";
 import { SlideStyle } from "./slide.style";
-import baseTheme from "../../../style/themes/base";
 
-const Slide = (props) => (
+export interface SlideProps {
```

Comment:
```suggestion
import { TagProps } from ""../../../__internal__/utils/helpers/tags/tags";
export interface SlideProps extends TagProps {
```

Since we have the tagComponent util for this component, I think we should extend the interface so the data-* attributes are recognised